CMakeLists.txt

```diff
@@ -43,6 +52,8 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ArborXConfig.cmake
 
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/src/ DESTINATION include
```

Comment:
I think you need to filter out files that only make sense for MPI, like all `ArborX_Distribut*`.

Reply:
I can give it a shot. Clearly, the pull request works without it.

Reply:
Right. It's not strictly necessary. Just wonder if it will cause confusion if one seed installed distributed headers but is unable to run MPI.

Reply:
So far, I hide everything behind a preprocessor switch so that the header can still be included in a non-`MPI` build. Maybe, not installing the files at all is a better solution.